### DISCUSSION TITLE
Arena stats.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#6882] [Suggestions]
CC already has [PvPstats](https://github.com/FrancescoBorzi/PvPstats) deployed for logging battleground match data, why not deploy [Arena Stats](https://github.com/azerothcore/arena-stats) now that arenas are available?

**Comment by Amrx2** [comment]
CC always had Arena Stats, just there is a link now on the website for it as well.

https://www.chromiecraft.com/en/arena-stats/
